<launch>
  <!-- CAMERAS -->
  <node name="camera2" pkg="usb_cam" type="usb_cam_node" output="screen">
    <param name="video_device" value="/dev/video1"/>
    <param name="image_width" value="640"/>
    <param name="image_height" value="480"/>
    <param name="camera_frame_id" value="camera2"/>
    <param name="io_method" value="mmap"/>
    <remap from="image_raw" to="camera2/camera_image"/>
  </node>
</launch>
